### PR TITLE
Don't show "No hints" output of Haskell's hlint

### PR DESCRIPTION
**Pull request by 2chilled**

### CODE REVIEW

File: autoload/neomake/makers/ft/haskell.vim [hunk 1]
```diff
@@ -109,6 +109,7 @@ function! neomake#makers#ft#haskell#HlintEntryProcess(entry) abort
     " Postprocess hlint output to make it more readable as a single line
     let a:entry.text = substitute(a:entry.text, '\v(Found:)\s*\n', ' | \1', 'g')
     let a:entry.text = substitute(a:entry.text, '\v(Why not:)\s*\n', ' | \1', 'g')
+    let a:entry.text = substitute(a:entry.text, 'No hints', '', 'g')
```
Comment (blueyed) [inline autoload/neomake/makers/ft/haskell.vim]: I think the regex pattern should be more strict, e.g. by matching the beginning/end of line etc.

/ cc @dkasak

**Comment by 2chilled** [comment]
@blueyed Made the pattern more strict

**Comment by blueyed** [comment]
Thanks!

**Comment by dkasak** [comment]
@blueyed Sorry, didn't notice the mention. Anyway, looks good to me.

**Pull request merged**